tutorials/printing.adoc

```diff
@@ -1,83 +1,218 @@
-testovací objekt jste si vybrali
-link:../stls/printing/20mm-box.stl[model kostky o přesných rozměrech
-20×20×10 mm]. Potřebujete ověřit, zda tiskárna skutečně vytiskne stejné
-rozměry, které jsou v modelu. Abyste ušetřili materiál, nastavte tisk
-tak, aby se vytiskly kostky 4 a každá z nich byla dutá, měla jednu
-obvodovou vrstvu a žádnou vrchní vrstvu (ve výsledku byste měli dostat 4
-hranaté kalíšky).
+K připojení do sítě ve které jsou tiskárny přístupné, využijte školní počítače.
+Na každé tiskárně lze nalézt štítek uvádějící číslo tiskárny. 
+Připojte se na svou tiskárnu nahrazením `X` za číslo vaší tiskárny v níže uvedené adrese.
 
-(Model je z https://www.thingiverse.com/thing:5573[The Essential
-Calibration Set], Public Domain.)
+Adresa tiskárny:: http://printerX.ucebny
 
-[[2-úkol-1-bod]]
-2. úkol (1 bod)
-^^^^^^^^^^^^^^^
+K přihlášení použijte níže uvedené přihlašovací údaje. Heslo sděluje cvičící.
 
-V předchozím úkolu jste již ověřili, že tiskárna tiskne správné rozměry.
-Teď byste rádi vynahradili své drahé polovičce, že jste v práci a
-testujete tiskárny místo toho, abyste byli doma. Proto se rozhodnete
-vytisknout link:../stls/printing/cuteocto.stl[model chobotnice] na
-památku. Bohužel je moc velká a nemáte tolik času. Proto musíte upravit
-rozměry a vytisknout ji o polovinu menší. Kdyby se něco pokazilo,
-vytisknete najednou raději hned 2 kopie. U chobotnice není nutné, aby
-byla dutá, nastavte tedy tisk tak, aby měla výplň alespoň 10%.
-Zajistěte, aby hlava chobotnice nebyla propadlá (použijte dostatečné
-množství perimetrů).
+Jméno:: student
+Heslo:: _sdělí cvičící_
 
-(Model je z https://www.thingiverse.com/thing:27053[Cute Octopus Says
-Hello], CC BY MakerBot.)
+Pokud se na tiskárnu nelze připojit, vyzkoušejte xref:../troubleshooting/printing.adoc#_na-tiskárnu-se-nelze-připojit[jiné možnosti].
 
-[[3-úkol-1-bod-povinný-úkol]]
-3. úkol (1 bod, povinný úkol)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+CAUTION: Po připojení se ujistěte, že se jedná o rozhraní pro vaši tiskárnu, která vám leží na stole. Ač sabotování tisku spolužákům se může zdát jako legrace, vězte, že není.
 
-Ukliďte po sobě. Kdo po sobě neuklidí, nedostane body.
+
+=== OctoPrint
+
+https://github.com/foosel/OctoPrint[OctoPrint] je jednoduché a přehledné webové rozhraní pro ovládání tiskáren připojených k počítači (v našem případě k Raspberry Pi). Umožňuje tiskárnu ručně ovládat, zobrazit přehledně její stav a odesílat tisková data na tiskárnu.
+
+.Snímek prohlížece s otevřeným OctoPrint rozhraním
+image::../images/printing/octoprint-main.png[OctoPrint]
+
+
+.Ruční ovládání v OctoPrint
+image::../images/printing/octoprint-control.png[Karta ručního ovládání, 250, float="right"]
+==== Ruční ovládání
+
+Na kartě ručního ovládání (Control) lze, jak název napovídá, ručně ovládat tiskárnu.
+Můžete posouvat tiskovou hlavu ve všech třech osách a také ovládat extruder. 
+
+První sloupec je velmi intuitivní, šipky představují směry, do kterých můžete tiskovou hlavu posunout.
```

Comment:
intuitivní se nedá měřit. zkusil bych nepsat takovéhle bezobsažné sdělení. Co třeba:

> Pomocí prvního sloupce můžete hýbat tiskovou hlavou ve směrech X (z vašeho pohledu doleva/doprava) a Y (z vašeho pohledu dozadu a dopředu (fyzicky se hýbe deska, ne hlava).

Reply:
BTW tisková hlava **není** pojem z obrázku výše. prosím sjednotit názvosloví.